plugins/michi/README.md

```
@@ -54,36 +56,109 @@ bash scripts/setup.sh
-| `/michi-multi-repo:propagate` | 各リポジトリへの仕様展開（並列実行） |
-| `/michi-multi-repo:dev-all` | 全リポジトリ実装（並列実行） |
+## 使用方法
+
+Michiスキルは2つの方法で使用できます：
+
+### 1. 自動発動（推奨）
+
+Claude が会話の文脈から自動的に適切なタイミングでスキルを発動します。
+
+**例**:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)